plugin/manager.go

```diff
@@ -117,9 +118,9 @@ func NewManager(config ManagerConfig) (*Manager, error) {
 		return nil, err
 	}
 
-	manager.blobStore, err = newBasicBlobStore(filepath.Join(manager.config.Root, "storage/blobs"))
+	manager.blobStore, err = local.NewStore(filepath.Join(manager.config.Root, "storage"))
```

Comment:
@dmcgowan Do you think this should use the metadata db wrapper like buildkit does with labels/leases support? Or what is the best for the migration path that would use contained contentstore in the future.

Reply:
Using the metadata db implementation is better since it makes cleanup a bit easier. I think its best to abstract those details a bit though so containerd can be used directly when available or in an embedded fashion.

Reply:
My intent here was to remove duplicate code.
I understand that we'll need to manage leases if/when we use the containerd managed content store, it's just not the intent of this PR.
I'm only using local.New so we don't have to adapt the current "basicBlobStore" implementation to the content.Store API foe the dist code. It's just nice that the on-disk layout is compatible.